### DISCUSSION TITLE
P-Tree Checkbox mode not working well with virtualscroll (17.9.0)

### DISCUSSION THREAD

**Discussion by jpeter06** [primefaces/community#1363] [PrimeNG]
Hello All, 
I have just discover that p-tree is not working well in **checkbox mod**e with **virtualscroll** . In the stackblitz example you can see than with the same data, it works ok without virtualscroll and wrong with it
[https://stackblitz.com/edit/3dvjpb](https://stackblitz.com/edit/3dvjpb?file=src%2Fapp%2Fdemo%2Ftree-checkbox-demo.html)

thanks for this wonderfull framework :-)

JPeter

**Comment by cetincakiroglu** [comment]
Hi,

As I can see it doesn't partially check the parent if a child is selected. Is this the issue? Could you please open an issue report in PrimeNG github repo with the example so we can address the issue in upcoming releases?
